src/model/dataTypes/MemberVariable.ts

```diff
@@ -5,21 +5,56 @@ import ListElement from "../modelInfo/ListElement";
 import SimpleElement from "../modelInfo/SimpleElement";
 import FilePath from "./FilePath";
 
+export const enum RelationshipType {
+  embeds_one = "embeds_one",
+  embeds_many = "embeds_many",
+  belongs_to = "belongs_to",
```

Comment:
Codacy found an issue: [Identifier 'belongs_to' is not in camel case.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5722975)